stylesheet.css

```diff
@@ -1,195 +1,421 @@
-html {     
-  box-sizing: border-box; 
-} 
-
 *, *:before, *:after {     
-  box-sizing: inherit; 
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
 }
 
-body {
-  background-color: white;
-  font-family: Verdana, Arial, sans-serif;
-  margin-left: 25%;
-  margin-right: 25%;
+ul {
+  list-style-type: none;
 }
 
 li {
-  display: inline; 
+  text-decoration: none;
 }
-
 /*-- link --*/
+a {
+  text-decoration: none;
+  color: initial;
+}
+
 a:link {
   text-decoration: none;
   color: black;
 }
 
 /*-- link hovered --*/
 a:hover {
-    background-color: blue;
-    color: white;
+  background-color: blue;
+  color: white;
+}
+
+/*body*/
+body {
+  font-family: 'Roboto', sans-serif;
+  font-size: 16px;
+  width: 100%;
+  margin: auto;
+  background-position: 0 5rem;
+  background: url("images/header.png") no-repeat;
+  box-sizing: border-box;
+  line-height: 1;
+}
+
+body section {
+  padding: 0 1.25rem;
+}
+
+body > header {
+  padding: 0 calc((100vw - 52rem)/2);
+  width: 100%;
+  background-color: white;
+  margin: auto;
+  height: 10vh;
+  position: fixed;
+  display: flex;
+  align-items: center;
+  font-family: "Montserrat";
+  z-index: 1;
+}
+
+body > header > img {
+  padding-left: 1rem;
+}
+
+body > header > nav > ul {
+  display: none;
+}
+
+/* main body footer*/
+body > footer {
+  background-color: blue;
+  color: white;
+  text-align: center;
+  font-family: 'Roboto';
```

Comment:
You don't need to overwrite `font-family` again if it's specified for the `body`, only in cases when it's different.